hudi-common/src/main/java/org/apache/hudi/common/table/view/AbstractTableFileSystemView.java

```diff
@@ -249,10 +249,10 @@ private void ensurePartitionLoadedCorrectly(String partition) {
    *
    * @param statuses List of File-Status
    */
-  private Stream<HoodieDataFile> convertFileStatusesToDataFiles(FileStatus[] statuses) {
+  private Stream<HoodieBaseFile> convertFileStatusesToDataFiles(FileStatus[] statuses) {
```

Comment:
convertFileStatusesToBaseFiles instead ? No strong opinion on these. I think it should be okay to leave it as is since the return type is indicative already.

Reply:
+1